csrankings.csv

```diff
@@ -5718,6 +5719,7 @@ Felix A. Wichmann,University of Tübingen,https://uni-tuebingen.de/en/faculties/
 Felix Brandt 0001,TU Munich,http://dss.in.tum.de/staff/brandt,fVnXdaQAAAAJ
 Felix C. Freiling,University of Erlangen–Nuremberg,https://www.cs1.tf.fau.de/person/felix-freiling,NOSCHOLARPAGE
 Felix Freitag,Polytechnic University of Catalonia,http://people.ac.upc.edu/felix,6wJ4ZdQAAAAJ
+Felix J. Herrmann,Georgia Institute of Technology,http://www.slim.eos.ubc.ca,hw23CTQAAAAJ

```

Comment:
In ECE; no obvious cross-appointment with CS.

Reply:
GT School of CSE faculty is listed here: https://www.cse.gatech.edu/people/faculty  and Felix's page is here: https://www.cse.gatech.edu/people/felix-herrmann  He  apparently didn't update his personal web page recently. He has joint appointment with our school (his office and lab is in the School of CSE). He can advise CS and CSE PhD students with this appointment. I wanted to be inclusive.